README.md

```diff
@@ -149,7 +207,20 @@ The ancestors have provided an example of one route; you do the other six!
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
-Replace this with your answer
+GET '/warriors'
+  * This route will show the emperor information about all his warriors.
+
+POST '/warriors'
+  * This route allows the emperor to add a new warrior to the Royal Military.
+
+PUT '/warriors/1'
+  * This route allows the emperor to update the warrior with id of 1 with all new content.
+
+PATCH '/warriors/1'
+  * This route allows the emperor to update the warrior with id of 1 with partially new content.
+
+DELETE '/warriors/1'
```

Comment:
which routes should i use to access the forms to create and update warriors? (new and edit routes)